python-avocado.spec

```diff
@@ -369,6 +377,10 @@ Again Shell code (and possibly other similar shells).
 %{_libexecdir}/avocado*
 
 %changelog
+* Thu Nov 11 2021 Cleber Rosa <cleber@redhat.com> - 92.0-2
+- Skip resultsdb plugin build on Fedora 36 due to broken resultsdb-api
+  release
+
 * Wed Oct 20 2021 Ana Guerrero Lopez <anguerre@redhat.com> - 92.0-2
```

Comment:
We have repeated twice the same version: 92.0-2

Reply:
Ooops, that is a mistake!